zinit-install.zsh

```diff
@@ -528,78 +528,81 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || {
 # $1 - plugin spec (4 formats: user---plugin, user/plugin, user, plugin)
 # $2 - plugin if $1 (i.e., user) given
 # $3 - if 1, then reinstall, otherwise only install completions that are not present
-.zinit-install-completions() {
+.zinit-install-completions () {
     builtin emulate -LR zsh ${=${options[xtrace]:#off}:+-o xtrace}
-    setopt nullglob extendedglob warncreateglobal typesetsilent noshortloops
+    setopt extended_glob no_short_loops null_glob typeset_silent warn_create_global
 
-    local id_as=$1${2:+${${${(M)1:#%}:+$2}:-/$2}}
-    local reinstall=${3:-0} quiet=${${4:+1}:-0}
-    (( OPTS[opt_-q,--quiet] )) && quiet=1
-    [[ $4 = -Q ]] && quiet=2
-    typeset -ga INSTALLED_COMPS SKIPPED_COMPS
-    INSTALLED_COMPS=() SKIPPED_COMPS=()
+    local id_as=$1${2:+${${${(M)1:#%}:+$2}:-/$2}} 
+    local reinstall=${3:-0} quiet=${${4:+1}:-0} 
+
+    (( OPTS[opt_-q,--quiet] )) && quiet=1 
+    [[ $4 = -Q ]] && quiet=2 
+
+    typeset -gaU INSTALLED_COMPS SKIPPED_COMPS
+    INSTALLED_COMPS=() SKIPPED_COMPS=() 
 
     .zinit-any-to-user-plugin "$id_as" ""
-    local user=${reply[-2]}
-    local plugin=${reply[-1]}
+    local user=${reply[-2]} 
+    local plugin=${reply[-1]} 
     .zinit-any-colorify-as-uspl2 "$user" "$plugin"
-    local abbrev_pspec=$REPLY
-
+    local abbrev_pspec=$REPLY 
     .zinit-exists-physically-message "$id_as" "" || return 1
-
-    # Symlink any completion files included in the plugin directory
-    typeset -a completions already_symlinked backup_comps
+    typeset -aU completions already_symlinked backup_comps
     local c cfile bkpfile
-    # The plugin == . is a semi-hack/trick to handle 'creinstall .' properly
+    nt () {
```

Comment:
whats `nt`?

Reply:
nt is just a random function name. It could be anything.

Reply:
Ok, stupid question: why not a speaking function name so one can actually know what it does by just reading the function name? reading the zinit code is hard enough even without random function names :-)

Reply:
@jankatins,

lol, I just took the name from the [zsh docs](https://zsh.sourceforge.io/Doc/Release/Expansion.html#Expansion). But yes, I will rename it.

<img width="1978" alt="Screenshot 2024-01-18 at 1 55 51 PM" src="https://github.com/zdharma-continuum/zinit/assets/10052309/0c3d80c6-bbf4-449f-bf9d-936f06784706">